weave

```diff
@@ -1266,17 +1266,37 @@ tls_arg() {
     PROXY_ARGS="$PROXY_ARGS $1 /home/weave/tls/$3.pem"
 }
 
+# TODO: Handle relative paths for args
+# TODO: Handle args with spaces
+host_arg() {
+  PROXY_HOST="$1"
+  if [ "$PROXY_HOST" != "${PROXY_HOST#unix://}" ]; then
+    host=$(dirname ${PROXY_HOST#unix://})
+    if [ "$host" = "${host#/}" ]; then
+      echo "When launching the proxy, unix sockets must be specified as an absolute path." >&2
+      exit 1
+    fi
+    if [ "$host" = "/var/run" ]; then
+      echo "WARNING: When launching the proxy, unix sockets directly in /var/run can lead to a bug with net namespace cleanup." >&2
+      echo "To work around this, put the unix socket in a subdirectory of /var/run, and symlink it into place." >&2
+      echo "For more info, see https://github.com/weaveworks/weave/issues/1455" >&2
```

Comment:
This is not helpful. "bug in namespace cleanup" means nothing to the vast majority of users. Plus it suggest that bug is in weave. And linking to an issue that is *closed* conveys the impression that we've actually fixed that bug.

Suggestion: describe the actual user visible problems this is causing.